### DISCUSSION TITLE
Cosmos DB - certyfikat DP-420

### DISCUSSION THREAD

**Discussion by mikolaj-jankowski** [AzureCommunityPL/group#457] [Pomoc]
Hej,
Czy zdawał ktoś ostatnio ten egzamin? Jestem ciekaw jak trudno było, za którym razem udało się zdać i ewentualnie jakie materiały pomogły w nauce.

**Comment by Radz1u** [comment]
Cześć,
tak. Zdałem w zeszłym tygodniu (2 podejście, pierwsze przerwane). Co do materiałów - oprócz praktyki codziennej, to polecam zapoznać się z materiałami na MS Learn - to co jest podpięte pod ten egzamin. Całkiem dobrze pokrywają to, co jest potem finalnie wymagane.

**Reply by mikolaj-jankowski** [reply]
@Radz1u a przerabiałeś może jakieś dumpy z pytaniami?

**Reply by Radz1u** [reply]
Nie, nigdy. Pominę moralitety, taki sposób przygotowania się do egzaminu nie działa w moim przypadku.

**Reply by mikolaj-jankowski** [reply]
Oki thx
